examples/bzlmod/MODULE.bazel

```diff
@@ -106,6 +109,9 @@ pip.parse(
     },
 )
 pip.parse(
+    entry_points = {
```

Comment:
@rickeylev, instead of `entry_point` macros, what do you think about the following extension API? You can see the `bzlmod` example `entry_point` test for how it works.